lib/javascript.js

```diff
@@ -1,6 +1,14 @@
 module.exports = {
+  plugins: [
+    'prefer-arrow',
+    'jsdoc',
+    'security',
+    'lodash',
+  ],
   extends: [
     'eslint:recommended',
+    'plugin:security/recommended',
+    'plugin:lodash/recommended',
```

Comment:
The recommended config seems like [it's super opinionated](https://github.com/wix/eslint-plugin-lodash/blob/master/src/index.js#L8-L51). According to the docs, it includes _all_ of the rules.

[These rules](https://github.com/wix/eslint-plugin-lodash#preference-over-native) in particular seem like they might cause problems in a lot of projects, esp. those that don't use lodash at all.

What do you think?

Reply:
I thought it would be nice, but then I read the rules, and don't see a reason to add it.